src/Map/src/Bridge/Leaflet/assets/src/map_controller.ts

```diff
@@ -63,13 +63,34 @@ export default class extends AbstractMapController<
         const marker = L.marker(position, { title, ...otherOptions, ...rawOptions }).addTo(this.map);
 
         if (infoWindow) {
-            this.createInfoWindow({ definition: infoWindow, marker });
+            this.createInfoWindowMarker({ definition: infoWindow, marker });
         }
 
         return marker;
     }
 
-    protected doCreateInfoWindow({
+    protected doCreatePolygon(definition: PolygonDefinition): L.Polygon {
+        const { points, title, rawOptions = {}, extra } = definition;
+
+        // Convertir les points en latlngs pour Leaflet
+        const latLngs = points.map((point) => [point.lat, point.lng]);
```

Comment:
I think this is not necessary, in `@types/leaflet` it looks like `L.polygon` accepts `[{lat: 1, lng: 2}]` as first parameter.